mono/metadata/class-init.c

```diff
@@ -5408,7 +5408,7 @@ mono_class_compute_gc_descriptor (MonoClass *klass)
 			for (int pos = 0; pos < weak_bitmap_nbits; ++pos) {
 				if (weak_bitmap [pos / BITMAP_EL_SIZE] & ((gsize)1) << (pos % BITMAP_EL_SIZE)) {
```

Comment:
while at it, let's fix that puppy too? how about using a variable?

Reply:
+1